relevanceai/api/batch/batch_insert.py

```diff
@@ -26,7 +26,7 @@ def insert_documents(
         bulk_fn: Callable = None,
         max_workers: int = 8,
         retry_chunk_mult: float = 0.5,
-        show_progress_bar: bool = False,
+        show_progress_bar: bool = True,
```

Comment:
Why should the progress bar be the default behaivour?

Reply:
I put it default because there it does not harm any other processes, but it provides significant benefit in knowing where your function is up to in downloading (especially for large datasets)

Reply:
Hmm - it's ifne. Let's default to False for now.